### ISSUE TITLE
Investigate tv bias lighting

### ISSUE THREAD

**Issue by pinkywafer** [pinkywafer/Home-Assistant_Config#15]

**Comment by CHBMB** [comment]
I've been looking at this stuff too, although limited by the fact I have a 4K setup and some of the hdmi capture devices don't work so well with HDR I'm currently holding off a bit.

But this *might* be what you're referring to.

https://hyperion-project.org/wiki/Main

**Comment by pinkywafer** [comment]
I've been kind of half looking at Hyperion... Does look like what I'm looking to do. 
Im also not sure how i plan to capture as i normally run Plex etc straight on the TV, but that may change!...
Not a high priority at the moment, but need to keep track of ideas and possible methods!

**Comment by CHBMB** [comment]
Yeah, it's not possible to capture the stream directly from the TV unfortunately.

You'd need a Roku or such like as far as I know.
